docs/source/API/core_objects/index.rst

```diff
@@ -1,12 +1,48 @@
-.. _coreobjects
-
 Core Objects
 ============
 
+The top-level object in a starfish workflow is the :ref:`Experiment`. It is composed of one or more
+:ref:`FieldOfView` objects, and a :ref:`Codebook`, which maps detected spots to the entities they
+target.
+
+Each :ref:`FieldOfView` consists of a set of Primary Images and optionally, Auxiliary images that
+may contain information on nuclei (often used to seed segmentation) or fiduciary beads (often used
```

Comment:
are these the only types of aux images?  Would clarify the wording to indicate more possibilities.